examples/aws/terraform/ha-autoscale-cluster/proxy_network.tf

```diff
@@ -72,6 +72,35 @@ resource "aws_security_group_rule" "proxy_ingress_allow_kube" {
   security_group_id = aws_security_group.proxy.id
 }
 
+// Permit inbound to Teleport mysql services
+resource "aws_security_group_rule" "proxy_ingress_allow_mysql" {
+  type              = "ingress"
+  from_port         = 3036
+  to_port           = 3036
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
```

Comment:
first time reading these example scripts. assuming we are using `0.0.0.0/0` all other places so i guess this is Ok. 

I think ideally this should be a variable with `["0.0.0.0/0"]` as default so that ppl can customize the CIDR range for better security.

Reply:
@greedy52 I've added variables for any references to `0.0.0.0/0`